pkg/configuration/configuration.go

```diff
@@ -50,8 +48,13 @@ const (
 
 	ClusterUnavailableDelay        = "cluster.unavailable.delay"
 	DefaultClusterUnavailableDelay = "60s"
+
+	identityProviderName        = "identity.provider"
+	DefaultIdentityProviderName = "rhd"
 )
 
+var log = logf.Log.WithName("configuration")
```

Comment:
not used 
```suggestion
```

Reply:
fixed here: cecef2d2b60d85e6e54793c4d68db1168ee8c449